plat/rockchip/rk3399/drivers/pmu/pmu.c

```diff
@@ -249,11 +667,20 @@ static int cores_pwr_domain_on(unsigned long mpidr, uint64_t entrypoint)
 	return 0;
 }
 
-static int cores_pwr_domain_off(void)
+static int cores_pwr_domain_off(uint32_t lvl, plat_local_state_t lvl_state)
```

Comment:
The cpu's suspend function may be more difficult than off,So higher level （clusters and system） suspend function is divided. But using divided function is clear。